include/fastdds/dds/core/policy/QosPolicies.hpp

```diff
@@ -466,8 +467,7 @@ class ReliabilityQosPolicy : public Parameter_t, public QosPolicy
     {
         return (this->kind == b.kind) &&
                (this->max_blocking_time == b.max_blocking_time) &&
-               Parameter_t::operator ==(b) &&
```

Comment:
Why did you remove this line?

Reply:
This was changed before the `QosPolicy::operator ==()` was corrected, and the I forgot about it. Good catch.